charts/super-agent/values.yaml

```diff
@@ -129,10 +116,6 @@ super-agent-deployment:
       # -- Here you can set New Relic' Super Agent configuration.
       # @default -- See `values.yaml` for examples
       content:
```

Comment:
is the endpoint below still honored? if not we should remove it from the values example:
```
        # opamp:
        #   # TODO The endpoint should be set automatically based on the licenseKey and on the nrStaging option if opamp.enable=true
        #   endpoint: https://opamp.service.newrelic.com/v1/opamp
        #   headers:
        #     # TODO this value should be automatically injected
        #     api-key: LICENSE_KEY
        #  # -- Note: To use the authentication configuration, ensure authSecret is configured.
        #   auth_config:
        #     token_url: "http://fake.com/oauth2/v1/token"
        #     client_id: "fake"
```